packages/celotool/src/lib/attestation.ts

```diff
@@ -15,19 +16,23 @@ export async function requestMoreAttestations(
   account: Address,
   txParams: CeloTransactionParams = {}
 ) {
-  await attestations
-    .approveAttestationFee(attestationsRequested)
-    .then((txo) => txo.sendAndWaitForReceipt(txParams))
-  await attestations
-    .request(phoneNumber, attestationsRequested)
-    .then((txo) => txo.sendAndWaitForReceipt(txParams))
+  const unselectedRequest = await attestations.getUnselectedRequest(phoneNumber, account)
```

Comment:
what does this do? Should I use it in the app?

Reply:
If the process quits for whatever reason after we requested, but not yet selected, requesting again will fail. So it would make sense for you to adopt it as well in the mobile app.